src/core/Microsoft.PowerToys.Settings.UI.Lib/ViewModels/KeyboardManagerViewModel.cs

```diff
@@ -129,28 +149,31 @@ public List<AppSpecificKeysDataModel> RemapShortcuts
 
         public ICommand EditShortcutCommand => _editShortcutCommand ?? (_editShortcutCommand = new RelayCommand(OnEditShortcut));
 
+        // Note: FxCop suggests calling ConfigureAwait() for the following methods,
+        // and calling ConfigureAwait(true) has the same behavior as not explicitly
+        // calling it (continuations are scheduled on the task-creating thread)
         private async void OnRemapKeyboard()
         {
-            await Task.Run(() => OnRemapKeyboardBackground());
+            await Task.Run(() => OnRemapKeyboardBackground()).ConfigureAwait(true);
```

Comment:
Just curious does fx cop actually warn on this particular await as well.  My understanding is that it wouldn't matter as the synchronization context isn't visible inside the task.run.

Reply:
Yeah, it's giving the [CA2007](https://docs.microsoft.com/en-us/dotnet/fundamentals/code-analysis/quality-rules/ca2007) warning. Looks like it's just suggesting an explicit declaration of the continuation context